README.md

```diff
@@ -4,6 +4,8 @@ This extension syncs data from Google's [Cloud Firestore](https://firebase.googl
 
 Out-of-the-box, Cloud Firestore provides no mechanism for full-text search on data. Syncing your Cloud Firestore data to Elastic App Search not only gives you a mechanism for full-text search on your data, it also lets you enjoy App Search's powerful relevance tuning features and search analytics data.
 
+**NOTE:** This extension is no longer maintained.  We encourage the community use the open and supported [connectors framework](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html#connectors-overview-framework) for Google Cloud Firestore integration development.
```

Comment:
@ppf2 - as written this makes it appear that the connector framework already supports Firestore as a connector, which it doesn't. I'd suggest something like:
> We encourage the community to use the open and supported [connectors framework](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html#connectors-overview-framework) to build an Elasticsearch connector for integration Google Cloud Firestore.

Reply:
Thanks and updated :)